README.md

```diff
@@ -134,6 +134,9 @@ The Azure MCP Server supercharges your agents with Azure context. Here are some
 - Deploy foundry models
 - List foundry model deployments
 
+### 🚀 Azure Managed Grafana
+- List Azure Managed Grafana
+
```

Comment:
Please also update https://github.com/Azure/azure-mcp/blob/main/README.md#-available-azure-mcp-servers

Reply:
We release twice a week.

Next release will be on Thur. :)